contracts/test/RoleBastion.t.sol

```diff
@@ -0,0 +1,90 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity ^0.8.0;
+
+import {DSTest} from "./utils/DSTest.sol";
+import {Vm} from "./utils/Vm.sol";
+import {getCore, getAddresses, FeiTestAddresses} from "./utils/Fixtures.sol";
+import {Core} from "../core/Core.sol";
+import {RoleBastion} from "../pods/RoleBastion.sol";
+import {TribeRoles} from "../core/TribeRoles.sol";
+
+// import "hardhat/console.sol";
+
+contract RoleBastionTest is DSTest {
```

Comment:
I'd like to see a test here that shows RoleBastion trying to grant privileged roles and failing